<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="prev_screen_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Vissza a kezdőoldalra</string>
   </property>
  </widget>
  <widget class="QLabel" name="sentences_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1010</y>
     <width>371</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>A két műben megtalálható mondatok száma.</string>
   </property>
  </widget>
  <widget class="QLabel" name="avg_sentence_label">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>1000</y>
     <width>391</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>A két műben megtalálható mondatok átlagos hossza.</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>530</y>
     <width>641</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="max_sentence_vbox"/>
  </widget>
  <widget class="QLabel" name="max_sentence_label">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>1010</y>
     <width>421</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>A két műben megtalálható leghosszabb mondat hossza.</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>530</y>
     <width>641</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="avg_sentences_vbox"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>530</y>
     <width>641</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="sentences_vbox"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>641</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vocab_vbox"/>
  </widget>
  <widget class="QLabel" name="vocab_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1050</y>
     <width>521</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>A két író összes feldolgozott művében előfurduló különböző szavak száma.</string>
   </property>
  </widget>
  <widget class="QLabel" name="all_words_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1030</y>
     <width>511</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>A két író összes feldolgozott művében előfurduló szavak száma.</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>641</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="all_words_vbox"/>
  </widget>
  <widget class="QComboBox" name="works_combo_box_1">
   <property name="geometry">
    <rect>
     <x>1430</x>
     <y>260</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="comparison_button">
   <property name="geometry">
    <rect>
     <x>1630</x>
     <y>280</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Összehasonlítás</string>
   </property>
  </widget>
  <widget class="QLabel" name="compare_label">
   <property name="geometry">
    <rect>
     <x>1430</x>
     <y>340</y>
     <width>331</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Művek összehasonlítása.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="works_combo_box_2">
   <property name="geometry">
    <rect>
     <x>1430</x>
     <y>300</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="next_screen_button">
   <property name="geometry">
    <rect>
     <x>1750</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>További elemzések</string>
   </property>
  </widget>
  <widget class="QLabel" name="moricz_less_than_100_label">
   <property name="geometry">
    <rect>
     <x>1673</x>
     <y>180</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="jokai_label">
   <property name="geometry">
    <rect>
     <x>1313</x>
     <y>150</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Jókai Mór</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="less_than_100_label">
   <property name="geometry">
    <rect>
     <x>1673</x>
     <y>120</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt; 100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="moricz_less_than_10_label">
   <property name="geometry">
    <rect>
     <x>1453</x>
     <y>180</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="moricz_less_than_50_label">
   <property name="geometry">
    <rect>
     <x>1563</x>
     <y>180</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="writer_label">
   <property name="geometry">
    <rect>
     <x>1313</x>
     <y>120</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Írók</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="moricz_label">
   <property name="geometry">
    <rect>
     <x>1313</x>
     <y>180</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Móricz Zsigmond</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="moricz_more_than_100_label">
   <property name="geometry">
    <rect>
     <x>1783</x>
     <y>180</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="less_than_50_label">
   <property name="geometry">
    <rect>
     <x>1563</x>
     <y>120</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt; 50</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="less_than_10_label">
   <property name="geometry">
    <rect>
     <x>1453</x>
     <y>120</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt; 10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="jokai_less_than_100_label">
   <property name="geometry">
    <rect>
     <x>1673</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="jokai_more_than_100_label">
   <property name="geometry">
    <rect>
     <x>1783</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="jokai_less_than_50_label">
   <property name="geometry">
    <rect>
     <x>1563</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="jokai_less_than_10_label">
   <property name="geometry">
    <rect>
     <x>1453</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="more_than_100_label">
   <property name="geometry">
    <rect>
     <x>1783</x>
     <y>120</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>100 &lt;= </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>1319</x>
     <y>140</y>
     <width>570</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>110</y>
     <width>570</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>1443</x>
     <y>115</y>
     <width>10</width>
     <height>91</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>1553</x>
     <y>114</y>
     <width>10</width>
     <height>91</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>200</y>
     <width>570</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>1663</x>
     <y>114</y>
     <width>10</width>
     <height>91</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>1773</x>
     <y>114</y>
     <width>10</width>
     <height>91</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>1315</x>
     <y>114</y>
     <width>10</width>
     <height>91</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>1885</x>
     <y>114</y>
     <width>10</width>
     <height>91</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>170</y>
     <width>570</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
